luda-editor/resources/sequence/testSequence.json

```diff
@@ -198,7 +198,7 @@
                     {
                         "id": "s-2-clip",
                         "start_at": {
-                            "milliseconds": 0
+                            "milliseconds": 1000
```

Comment:
Is it intended to include this change?

Reply:
Yes, is it. not important, but for test.